cli/cli.js

```diff
@@ -183,7 +183,12 @@ program
       path = AVA_BIP32_PREFIX + "/" + path;
       console.error("Getting public key for path", path);
       requestLedgerAccept();
+
+      if(options.speculos) {
```

Comment:
Let's stick with consistent spacing here: `if (`
```suggestion
      if (options.speculos) {
```